dc_utils/tests/test_forms.py

```diff
@@ -0,0 +1,51 @@
+from django.test import TestCase
+from dc_signup_form.forms import MailingListSignupForm
+from dc_signup_form.constants import MAILING_LIST_FORM_PREFIX
+
+def add_data_prefix(prefix, data):
+    return {
+        "{}-{}".format(prefix, k): v
+        for k, v in data.items()
+        if not str(v).startswith(k)
+    }
+
+
+class TestForms(TestCase):
+    def test_mailing_list_form_valid(self):
+        data = add_data_prefix(MAILING_LIST_FORM_PREFIX, {
+            'full_name': 'Chad Fernandez',
+            'email': 'chad.fernandez@democracyclub.org.uk',
+            'source_url': 'http://foo.bar/baz',
+            'main_list': True,
+        })
+
+        form = MailingListSignupForm(data)
+        if not form.is_valid():
+            print(form.errors)
```

Comment:
I guess this was useful for testing, but it might be better to do `assert form.errors = []` (or whatever the non-error value is), so that if it fails in future we at least get the errors displayed